contextx/runner.go

```diff
@@ -20,18 +20,16 @@ func (f RunnerFunc) Run(ctx context.Context) {
 	f(ctx)
 }
 
-// EmptyRunner is a Runner that does nothing for testing purposes only.
-func EmptyRunner() RunnerFunc {
+// Empty is a Runner that does nothing for testing purposes only.
+func Empty() RunnerFunc {
```

Comment:
contextx.Empty() looks weird when you use it, I prefer contextx.EmptyRunner() (same for MultiRunner)